blueprints/movie/service.py

```diff
@@ -46,3 +45,26 @@ def svc_post(payload):
     result = do_query(sql, params)
 
     return result
+
+
+def svc_put(payload, id):
+    """
+    A PUT Service
+    """
+    title = payload["title"]
+    description = payload["description"]
+    year = payload["movie_year"]
+    rating = payload["rating"]
+    runtime = payload["runtime"]
+    votes = payload["votes"]
+    revenue = payload["revenue"]
+    metascore = payload["metascore"]
+    created_at = payload["created_at"]
+
+    # SQL statement
+    sql = f"UPDATE {SCHEMA_NAME}.{MOVIE} SET title = %(title)s, description = %(description)s, movie_year = %(movie_year)s, rating = %(rating)s, runtime = %(runtime)s, votes = %(votes)s, revenue = %(revenue)s, metascore = %(metascore)s, created_at = %(created_at)s WHERE movie_id = %(id)s RETURNING *;"
+    params = {"title": title, "description": description, "movie_year": year, "rating": rating, "runtime": runtime, "votes": votes, "revenue": revenue, "metascore": metascore, "created_at": created_at, "id": id}
```

Comment:
Why dictionary is not prettified? Make it readable. VScode can format it for ya! Or use jsonlint to format it. Google it.